src/fluidObjects/diceRoller/DiceRollerView.tsx

```diff
@@ -29,20 +30,16 @@ export const DiceRollerView = (props: IAppProps) => {
             // When the view dismounts remove the listener to avoid memory leaks
             props.model.off("diceRolled", onDiceRolled);
         };
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [diceValue]);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [props]);
```

Comment:
We were just doing this wrong?

Reply:
how often do we want to re-evaluate this useEffect? Every time the diceValue changes? Or when the component props change?